### ISSUE TITLE
Function arent working

### ISSUE THREAD

**Issue by benjieng4** [BillPetti/baseballr#310]
I have been trying to use the function "fg_batting_leaders" for my projects. However, I get the following error message on Rstudio

"Error in fg_batting_leaders(x = 2022, y = 2022, qual = 200) : 
  could not find function "fg_batting_leaders"

I got a new computer and on my old computer everything was working fine. This function and the pitcher function won't work.

Any suggestions?

**Comment by ryanb519** [comment]
I'm getting the same thing. FanGraphs just made a bunch of changes to their leaderboards (https://blogs.fangraphs.com/weve-updated-our-major-league-leaderboards-interface/), so I assume this broke the fg-related functions in BaseballR :/

**Comment by rdelrossi** [comment]
@benjieng4, try `fg_batter_leaders(startseason = 2022, endseason = 2022, qual = 200)`.

**Comment by saiemgilani** [comment]
> @benjieng4, try `fg_batter_leaders(startseason = 2022, endseason = 2022, qual = 200)`.

correct, but this is a new change in baseballr version 1.6.0 (soon to be pushed to CRAN, in the meanwhile run):

```r
remotes::install_github("BillPetti/baseballr")
```

then restart your R session and try with the quoted command

**Comment by saiemgilani** [comment]
The function is **`fg_batter_leaders()`** (and `fg_pitcher_leaders()`), not `fg_batting_leaders()` — the `could not find function "fg_batting_leaders"` error is simply because that name has never existed in the package. On the new machine the package most likely wasn't installed (or an old build was loaded).

Verified on `master`: `fg_batter_leaders(startseason = "2023", endseason = "2023", qual = "200")` returns 362 rows. (FanGraphs itself was also down behind a Cloudflare challenge for a while — that's fixed in 2.0.0 via the okhttp User-Agent, PRs #403/#405.)

```r
remotes::install_github("BillPetti/baseballr")
fg_batter_leaders(startseason = "2022", endseason = "2022", qual = "200")
```

Closing — please reopen if the correctly-named `fg_batter_leaders()` fails after reinstalling.
